web/thesauruses/c/17/strings.json

```diff
@@ -0,0 +1,302 @@
+      "not-implemented":"true",
+      "name": "Is string all alphanumeric characters?"
+    },
+    "is_decimal": {
+      "not-implemented":"true",
+      "name": "Is string a decimal number?"
+    },
+    "is_all_whitespaces": {
+      "not-implemented":"true",
+      "name": "Is string all whitespace characters?"
+    },
+    "is_all_uppercase": {
+      "not-implemented":"true",
+      "name": "Is string all uppercase characters?"
+    },
+    "is_all_lowercase": {
+      "not-implemented":"true",
+      "name": "Is string all lowercase characters?"
+    },
+    "is_in_titlecase": {
+      "not-implemented":"true",
+      "name": "Is string formatted in title case?"
+    },
+    "does_substring_exist": {
+      "not-implemented": "true",
+      "name": "Does a substring exist in a string?"
+    },
+    "find_start_index_of_substring": {
+      "not-implemented": "true",
+      "name": "Find index of where a substring starts"
+    },
+    "find_start_index_of_additional_substring": {
+      "not-implemented": "true",
+      "name": "Find index of an additional substring (or starting at another index)"
+    },
+    "find_start_index_of_substring_from_end": {
+      "not-implemented": "true",
+      "name": "Find substring index starting at end"
+    },
+    "count_occurrences_of_substring": {
+      "not-implemented": "true",
+      "name": "Find number of occurences of a substring"
+    },
+    "get_leftmost_characters": {
+      "not-implemented": "true",
+      "name": "Get a specified number of characters from the left"
+    },
+    "get_rightmost_characters": {
+      "not-implemented": "true",
+      "name": "Get a specified number of characters from the right"
+    },
+    "get_substring_from_start_and_end_index": {
+      "not-implemented": "true",
+      "name": "Return a substring from a string based on starting and ending indices"
+    },
+    "get_substring_from_start_index_and_length": {
+      "not-implemented": "true",
+      "name": "Return a substring from a string based on starting index and size of substring"
+    },
+    "convert_to_uppercase": {
+      "code": [
+        "#include<stdio.h>",
+        "#include<string.h>",
+        "char str[100];",
+        "gets(str);",
+        "printf(\"%s\",strupr(str);"
+      ],
+      "name": "Convert string to all uppercase"
+    },
+    "convert_to_lowercase": {
+      "code": [
+        "#include<stdio.h>",
+        "#include<string.h>",
+        "char str[100];",
+        "gets(str);",
+        "printf(\"%s\",strlwr(str);"
+      ],
+      "name": "Convert string to all lowercase"
+    },
+    "convert_to_title_case": {
+      "not-implemented": "true",
+      "name": "Convert string to title case"
+    },
+    "capitalize_string": {
+      "not-implemented": "true",
+      "name": "Capitalize first letter of a string"
+    },
+    "remove_whitespace": {
+      "not-implemented": "true",
+      "comment":"isspace() is available for custom implementation",
```

Comment:
I think you can remove this. The idea of a thesaurus wouldn't suggest how to make your own.